pkgs/development/python-modules/sphinxcontrib-excel-table/default.nix

```diff
@@ -0,0 +1,30 @@
+{ lib
+, buildPythonPackage
+, fetchPypi
+, sphinx
+, openpyxl
+, nose
+}:
+
+buildPythonPackage rec {
+  pname = "sphinxcontrib-excel-table";
+  version = "1.0.8";
+
+  src = fetchPypi {
+    inherit pname version;
+    hash = "sha256:1q79byn3k3ribvwqafbpixwabjhymk46ns8ym0hxcn8vhf5nljzd";
+  };
+
+  buildInputs = [ nose ];
```

Comment:
`nose` is a `checkInputs`. `nose` should be removed as there are no tests (`Ran 0 tests in 0.000s`).

Please add `doCheck = false;` and `pythonImportsCheck`.

Reply:
Done (`pythonImportsCheck` was already there)